wcivf/apps/elections/templates/elections/includes/inline_elections_nav_list.html

```diff
@@ -41,7 +42,7 @@ <h3>
                     {{ election_group.grouper|naturalday:"l j F Y"|title }}
                 </h3>
                 <p>
-                    {% blocktrans trimmed with num_ballots=election_group.list|length|apnumber ballots_pluralised=election_group.list|pluralize %}
+                    {% blocktrans trimmed with num_ballots=election_group.list|uncancelled_ballots|length|apnumber ballots_pluralised=election_group.list|pluralize %}
                         You will have {{ num_ballots }} ballot paper{{ ballots_pluralised }} to fill out.
                     {% endblocktrans %}
```

Comment:
If all my elections are cancelled, is this going to tell me "you will have 0 ballot papers to fill out"? Do we want a special case for that?